### ISSUE TITLE
scil_assign_custom_color_to_tractogram colormap jpg output possible?

### ISSUE THREAD

**Issue by mdesco** [scilus/scilpy#628]
@frheault @CHrlS98 some of our users use our scripts and end-up having to generate a pseudo-accurate colormap png image from internet to create figures where they show a bundle with a certain colormap (metric, or pvalue or whatever). It would be extremely useful to add the option to output a small colormap figure. Do you think this would be feasible easily?

**Comment by CHrlS98** [comment]
Yes, easy and I'm pretty sure I have some code doing just that somewhere. I'll look into it.
